pkg/cmd/addon/enable/cmd.go

```diff
@@ -12,10 +12,24 @@ import (
 )
 
 var example = `
-# Enable addon on to the given managed clusters of the specify namespace
+## Application Manager
+
+# Enable application-manager addon on the given managed clusters in the specified namespace
 %[1]s addon enable --names application-manager --namespace namespace --clusters cluster1,cluster2
-# Enable application-manager addon specified clusters
+# Enable application-manager addon for specified clusters
 %[1]s addon enable --names application-manager --clusters cluster1,cluster2
+
+## Policy Framework
```

Comment:
Could you please add an example with the annotation for installing on the Hub?

Reply:
I was thinking of leaving it off since it's a somewhat special case, but it would be good to have documented in the CLI. I'll add it in.